apps/judicial-system/web/src/components/DefenderInfo/DefenderInput.tsx

```diff
@@ -0,0 +1,183 @@
+  ] = useState<string>('')
+
+  const options = useMemo(
+    () =>
+      lawyers.map((l: Lawyer) => ({
+        label: `${l.name}${l.practice ? ` (${l.practice})` : ''}`,
+        value: l.email,
+      })),
+
+    [lawyers],
+  )
+
+  const onChange = useCallback(
+    async (selectedOption: ValueType<ReactSelectOption>) => {
+      let updatedLawyer = {
+        defenderName: '',
+        defenderNationalId: '',
+        defenderEmail: '',
+        defenderPhoneNumber: '',
+      }
+
+      if (selectedOption) {
+        const {
+          label,
+          value,
+          __isNew__: defenderNotFound,
+        } = selectedOption as ReactSelectOption
+
+        onDefenderNotFound(defenderNotFound || false)
+
+        const lawyer = lawyers.find(
+          (l: Lawyer) => l.email === (value as string),
+        )
+
+        updatedLawyer = {
+          defenderName: lawyer ? lawyer.name : label,
+          defenderNationalId: lawyer ? lawyer.nationalId : '',
+          defenderEmail: lawyer ? lawyer.email : '',
+          defenderPhoneNumber: lawyer ? lawyer.phoneNr : '',
+        }
+      }
+
+      await updateCase(workingCase.id, updatedLawyer)
+      setWorkingCase({ ...workingCase, ...updatedLawyer })
+    },
+    [lawyers, setWorkingCase, workingCase, updateCase, onDefenderNotFound],
+  )
+
+  return (
+    <>
+      <Box marginBottom={2}>
+        <Select
+          name="defenderName"
+          icon="search"
+          options={options}
+          label={formatMessage(m.nameLabel, {
+            sessionArrangements: workingCase.sessionArrangements,
+          })}
+          placeholder={formatMessage(m.namePlaceholder)}
+          value={
+            workingCase.defenderName
+              ? {
+                  label: workingCase.defenderName ?? '',
+                  value: workingCase.defenderEmail ?? '',
+                }
+              : null
+          }
+          onChange={onChange}
+          filterConfig={{ matchFrom: 'start' }}
+          isCreatable
+          disabled={disabled}
+        />
+      </Box>
+      <Box marginBottom={2}>
+        <Input
+          data-testid="defenderEmail"
+          name="defenderEmail"
+          autoComplete="off"
+          label={formatMessage(m.emailLabel, {
+            sessionArrangements: workingCase.sessionArrangements,
+          })}
+          placeholder={formatMessage(m.emailPlaceholder)}
+          value={workingCase.defenderEmail}
+          errorMessage={emailErrorMessage}
+          hasError={emailErrorMessage !== ''}
+          disabled={disabled}
+          onChange={(event) =>
+            removeTabsValidateAndSet(
+              'defenderEmail',
+              event.target.value,
+              ['email-format'],
+              workingCase,
+              setWorkingCase,
+              emailErrorMessage,
+              setEmailErrorMessage,
+            )
+          }
+          onBlur={(event) =>
+            validateAndSendToServer(
```

Comment:
I thought we had removed this validateAndSendToServer 🤔 

Please consider using the `validateAndSendToServer` we created in the useCase hook.